### DISCUSSION TITLE
Domain transfer in problem

### DISCUSSION THREAD

**Discussion by typhondev** [vercel/community#1692] [Help] [answered]
### Summary

Hello, I'd like to use *.ai domain but I can't create that domain on vercel.
Also it is impossible to transfer in and it gives this error: `We do not support ".ai" TLD yet.`
Please let me know if there is any timeline or plan for ai TLD support in vercel.

**Comment by amyegan** [comment, marked as answer]
I don't have an ETA on `.ai` availability. But you can [add a custom domain to your project](https://vercel.com/docs/concepts/projects/domains/add-a-domain#) by using A and CNAME records or by changing nameservers. There is no need to transfer the domain to get it working for your site 🙂
